pica2mods-xslt/pom.xml

```diff
@@ -17,24 +16,6 @@
   </properties>
   <build>
     <plugins>
-      <plugin>
```

Comment:
Without this plugin-configuration Bamboo rejected the deployment to the Sonatype Maven Repository